src/components/PetThumbnail.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const PetThumbnail = ({ thumbnail, thumbnailAltText }) =>
+  thumbnail ? (
+    <img className="dg_image left" src={thumbnail} alt={thumbnailAltText} />
```

Comment:
we do not need `left` on the thumbnail. The grid will be responsible for the positioning of the image. 

But we should add a style with max-width = 100% for now, so that the image responds to smaller screens.

Reply:
ignore the max width thing.

Reply:
done

Reply:
oh well I had done this before you said ignore. Should I remove or just leave it?